js/modules/settings.js

```diff
@@ -6,6 +6,8 @@ const LAST_PROCESSED_INDEX_KEY = 'attachmentMigration_lastProcessedIndex';
 const IS_MIGRATION_COMPLETE_KEY = 'attachmentMigration_isComplete';
 
 // Public API
+exports.READ_RECEIPT_CONFIGURATION_SYNC = 'read-receipt-configuration-sync';
```

Comment:
Trying to centralize all setting names here. Would have added the getter/setter too but since this module requires IDB `connection`, it would have blown the scope of this issue, IMO.